### ISSUE TITLE
Need some advice on network training losses

### ISSUE THREAD

**Issue by Merical** [jiexiong2016/GCNv2_SLAM#5]
Hi. I'm working on a program to extract keypoints and descriptors with pytorch. I'm really interested in your work and try to reimplement it in python. But I'm struggling with the loss function. May I ask for some advice or demo codes to help me complete the loss function for keypoints and descriptors? Thanks~

**Comment by jiexiong2016** [comment]
Hi, thank you for your interest in GCNv2! The paper is currently under review, we plan to release the training code after the review process is completed.

Sincerely,
Jiexiong

**Comment by ChLee98** [comment]
> Hi. I'm working on a program to extract keypoints and descriptors with pytorch. I'm really interested in your work and try to reimplement it in python. But I'm struggling with the loss function. May I ask for some advice or demo codes to help me complete the loss function for keypoints and descriptors? Thanks~

Hi, have you completed the loss function for keypoints and descriptors ? I also have some problems about the loss function.

**Comment by match08** [comment]
publish traking GCN codes？

**Comment by liandexu186** [comment]
> > Hi. I'm working on a program to extract keypoints and descriptors with pytorch. I'm really interested in your work and try to reimplement it in python. But I'm struggling with the loss function. May I ask for some advice or demo codes to help me complete the loss function for keypoints and descriptors? Thanks~
> 
> Hi, have you completed the loss function for keypoints and descriptors ? I also have some problems about the loss function.

me too
